vips/vip-442/bscmainnet.ts

```diff
@@ -0,0 +1,244 @@
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            marketSpec.vToken.underlying.address,
+            [CHAINLINK_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+      },
+
+      // Add Market
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_supportMarket(address)",
+        params: [marketSpec.vToken.address],
+      },
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setMarketSupplyCaps(address[],uint256[])",
+        params: [[marketSpec.vToken.address], [marketSpec.riskParameters.supplyCap]],
+      },
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setMarketBorrowCaps(address[],uint256[])",
+        params: [[marketSpec.vToken.address], [marketSpec.riskParameters.borrowCap]],
+      },
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setCollateralFactor(address,uint256)",
+        params: [marketSpec.vToken.address, marketSpec.riskParameters.collateralFactor],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setAccessControlManager(address)",
+        params: [ACCESS_CONTROL_MANAGER],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setProtocolShareReserve(address)",
+        params: [PROTOCOL_SHARE_RESERVE],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: [REDUCE_RESERVES_BLOCK_DELTA],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "_setReserveFactor(uint256)",
+        params: [marketSpec.riskParameters.reserveFactor],
+      },
+
+      // Mint initial supply
+      {
+        target: VTREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [marketSpec.vToken.underlying.address, marketSpec.initialSupply.amount, NORMAL_TIMELOCK],
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [marketSpec.vToken.address, marketSpec.initialSupply.amount],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "mintBehalf(address,uint256)",
+        params: [marketSpec.initialSupply.vTokenReceiver, marketSpec.initialSupply.amount],
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [marketSpec.vToken.address, 0],
+      },
+      {
+        target: VTREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [marketSpec.vToken.underlying.address, SOL_INCENTIVES, VANGUARD_TREASURY],
```

Comment:
We have to transfer 5,000 USDT to the Vanguard treasury, not 5,000 SOL

Reply:
Ah, my bad. fixed, https://github.com/VenusProtocol/vips/commit/4997aaee6d2f4433ef205c49fb6761f9bd951cec